File: Java/src/irrgarten/main/TextMain.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package irrgarten.main;

import irrgarten.UI.TextUI;
import irrgarten.controller.ControllerP3;
import irrgarten.Game;

/**
 *
 * @author jorge
 */
public class TextMain {
    public static void main(String args[]){
        Game game = new Game(2);
        TextUI view = new TextUI();
        ControllerP3 controller = new ControllerP3(game,view);
        
        controller.play();
    }
}

File: Ruby/TextMain.rb
#encoding: UTF-8

require_relative 'Game'
require_relative 'Controller'
require_relative 'TextUI'

module Irrgarten
	module TextMain
		class TextMain
			def initialize
			end
			
			def main
				game = Game.new(2)
				view = UI::TextUI.new()
				controller = Control::Controller.new(game,view)
			
				controller.play
			end
		end
	end
end

Irrgarten::TextMain::TextMain.new.main
#juego=TextMain.new
#juego.main
